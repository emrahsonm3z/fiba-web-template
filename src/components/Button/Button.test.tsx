import { shallow } from 'enzyme'
import React from 'react'

import { Button } from '.'

it('runs correctly', () => {
  const wrapper = shallow(<Button />)

  expect(wrapper).toMatchSnapshot()
})

describe('Hello, Enzyme!', () => {
  it('renders', () => {
    const wrapper = shallow(<Button />)
    expect(wrapper.html()).toMatch(/Click/)
  })

  it('renders snapshots, too', () => {
    const wrapper = shallow(<Button />)
    expect(wrapper).toMatchSnapshot()
  })
})
