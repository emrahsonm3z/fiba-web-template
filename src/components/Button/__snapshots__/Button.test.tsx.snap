// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello, Enzyme! renders snapshots, too 1`] = `
<button
  type="button"
>
  Click
</button>
`;

exports[`runs correctly 1`] = `
<button
  type="button"
>
  Click
</button>
`;
